src/core-plugins/CoreCommands.ts

```diff
@@ -5,6 +5,9 @@ import { Plugin } from "../entities/Plugin";
 import { getByProps, Locale, MessageActions } from "../metro";
 import { DebugInfo } from "../utils/debug/DebugInfo";
 import { makeAsyncEval } from "../utils/misc";
+import { exists, externalStorageDirectory } from "../native/fs";
+
+const customBundle: boolean = AliuFS.exists(externalStorageDirectory + "/AliucordRN/Aliucord.js.bundle");
```

Comment:
You shouldn't need to put explicit types here

Reply:
thought ts was all about explicit static types